File: 1791-richest-customer-wealth/richest-customer-wealth.ts
function maximumWealth(accounts: number[][]): number {

    let wealth = Number.MIN_SAFE_INTEGER;
    let sum =0;

    for(let i=0;i<accounts.length;i++){
        sum = 0;
        for(let j=0;j<accounts[0].length;j++){
            sum = sum + accounts[i][j];
        }

        wealth = Math.max(wealth,sum);
    }

    return wealth;
    
};

File: 1791-richest-customer-wealth/richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {

    int wealth = Integer.MIN_VALUE;
    int sum =0;

    for(int i=0;i<accounts.length;i++){
        sum = 0;
        for(int j=0;j<accounts[0].length;j++){
            sum = sum + accounts[i][j];
        }
        wealth = Math.max(wealth,sum);
    }

    return wealth;
        
    }
}